webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -227,39 +231,71 @@ object FatalException {
  */
 object UserLogout {
 
-  def cleanUpSession(session: LiftSession, endCause: String) = {
-    SecurityContextHolder.getContext.getAuthentication match {
+  val logoutActions = Ref.make(Chunk[LogoutPostAction]()).runNow
```

Comment:
a storage for action that plugins want to register post logout, like a redirect for SLO (single logout).
Action can return an URI, which is understood as the redirect. Only the first URI is used (because, well, can't do anything else).